data/core/macros/names.cfg

```diff
@@ -139,9 +139,9 @@ centre_v=na|lla|na|ry|li|ni|ri|rae|bry
 >>
 #enddef
 
-#define KHALIFATE_NAMES
```

Comment:
I suggest adding `KHALIFATE_NAMES` to deprecated-utils.cfg as an alias, to maintain backwards compatibility with addons.